### PR TITLE
Include reference to added reference to the The EPUB Accessibility exemption property

### PR DESCRIPTION
**Pull request by clapierre**

### CODE REVIEW

**Comment by mattgarrish** [comment]
I'd also link the three references to the property in the table -- the ones with the text "a11y:exemption with the text string ...".

**Comment by mattgarrish** [comment]
I was kind of surprised to find that the values didn't have linkable ids. It's not a big table of values, but still.

Maybe we should update that note first to add them, since republishing notes doesn't require any special process.

**Comment by clapierre** [comment]
I noticed that too.  If you add the Id's to the Note I will update all of these references.

File: crosswalk/index.html [hunk 1]
```diff
@@ -279,25 +280,25 @@ <h2>EPUB and ONIX</h2>
 						</td>
 					</tr>
                     
-					<tr>
+					<tr id="eaa-microenterprise">
```
Comment (mattgarrish) [inline crosswalk/index.html:283]: We don't need IDs here, although maybe they don't hurt anything. It was the note that we needed to deep link into and that's done now. For this document, it's the hyperlinks that need to be fixed to be more specific. I'll suggest the changes for each. I'll create suggestions to remove the IDs but feel free to ignore the suggestions if you want to keep them.

File: crosswalk/index.html [hunk 2]
```diff
@@ -279,25 +280,25 @@ <h2>EPUB and ONIX</h2>
...
 						<td>
-							<p><code>a11y:exemption</code> with the text string<br/><code>eaa-microenterprise</code></p>
+							<p><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#exemption-property">a11y:exemption</a></code> with the text string<br/><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#exemption-values">eaa-microenterprise</a></code></p>
```
Comment (mattgarrish) [inline crosswalk/index.html:285]:
```suggestion
							<p><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#exemption-property">a11y:exemption</a></code> with the text string<br/><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#eaa-microenterprise">eaa-microenterprise</a></code></p>
```

File: crosswalk/index.html [hunk 3]
```diff
@@ -279,25 +280,25 @@ <h2>EPUB and ONIX</h2>
...
 						</td>
 						<td>
 							<p><a href="https://ns.editeur.org/onix/en/196/75">Code 75</a>: EAA exception 1 - Micro-enterprises</p>
 						</td>
 					</tr> 
-					<tr>
+					<tr id="eaa-disproportionate-burden">
 						<td>
-							<p><code>a11y:exemption</code> with the text string<br/><code>eaa-disproportionate-burden</code></p>
+							<p><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#exemption-property">a11y:exemption</a></code> with the text string<br/><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#exemption-values">eaa-disproportionate-burden</a></code></p>
```
Comment (mattgarrish) [inline crosswalk/index.html:293]:
```suggestion
							<p><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#exemption-property">a11y:exemption</a></code> with the text string<br/><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#eaa-disproportionate-burden">eaa-disproportionate-burden</a></code></p>
```

File: crosswalk/index.html [hunk 4]
```diff
@@ -279,25 +280,25 @@ <h2>EPUB and ONIX</h2>
...
 						<td>
 							<p><a href="https://ns.editeur.org/onix/en/196/76">Code 76</a>: EAA exception 2 - Disproportionate burden</p>
 						</td>
 					</tr>
-					<tr>
-						<td>
-							<p><code>a11y:exemption</code> with the text string<br/><code>eaa-fundamental-alteration</code></p>
+					<tr id="eaa-fundamental-alteration">
+						<td> 
+							<p><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#exemption-property">a11y:exemption</a></code> with the text string<br/><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#exemption-values">eaa-fundamental-alteration</a></code></p>
```
Comment (mattgarrish) [inline crosswalk/index.html:301]:
```suggestion
							<p><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#exemption-property">a11y:exemption</a></code> with the text string<br/><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#eaa-fundamental-alteration">eaa-fundamental-alteration</a></code></p>
```

File: crosswalk/index.html [hunk 5]
```diff
@@ -279,25 +280,25 @@ <h2>EPUB and ONIX</h2>
...
+							<p><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#exemption-property">a11y:exemption</a></code> with the text string<br/><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#exemption-values">eaa-disproportionate-burden</a></code></p>
 						</td>
 						<td>
 							<p><a href="https://ns.editeur.org/onix/en/196/76">Code 76</a>: EAA exception 2 - Disproportionate burden</p>
 						</td>
 					</tr>
-					<tr>
-						<td>
-							<p><code>a11y:exemption</code> with the text string<br/><code>eaa-fundamental-alteration</code></p>
+					<tr id="eaa-fundamental-alteration">
```
Comment (mattgarrish) [inline crosswalk/index.html:299]:
```suggestion
					<tr>
```

File: crosswalk/index.html [hunk 6]
```diff
@@ -279,25 +280,25 @@ <h2>EPUB and ONIX</h2>
...
 						<td>
-							<p><code>a11y:exemption</code> with the text string<br/><code>eaa-microenterprise</code></p>
+							<p><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#exemption-property">a11y:exemption</a></code> with the text string<br/><code><a href="https://www.w3.org/TR/epub-a11y-exemption/#exemption-values">eaa-microenterprise</a></code></p>
 						</td>
 						<td>
 							<p><a href="https://ns.editeur.org/onix/en/196/75">Code 75</a>: EAA exception 1 - Micro-enterprises</p>
 						</td>
 					</tr> 
-					<tr>
+					<tr id="eaa-disproportionate-burden">
```
Comment (mattgarrish) [inline crosswalk/index.html:291]:
```suggestion
					<tr>
```

File: crosswalk/index.html (hunk 1 above)
Comment (mattgarrish) [inline crosswalk/index.html:283]:
```suggestion
					<tr>
```

**Review by mattgarrish — CHANGES_REQUESTED** [review]

**Comment by clapierre** [comment]
Thanks @mattgarrish, all should now be addressed.

**Pull request merged**